test/components/blankslate_component_test.rb

```diff
@@ -54,17 +54,17 @@ def test_renders_a_blankslate_component_with_an_icon
                     title: "Title"
                   ))
 
-    assert_selector(".blankslate-icon[height=32]")
+    assert_selector(".blankslate-icon[height=24]")
   end
 
   def test_renders_a_blankslate_component_with_an_icon_with_a_custom_size
     render_inline(Primer::BlankslateComponent.new(
                     icon: "octoface",
-                    icon_size: :large,
+                    icon_size: :medium,
```

Comment:
Dod you check to see if any blankslate components use the `large` option?

Reply:
I did, there isn't 
![image](https://user-images.githubusercontent.com/54012/117180479-12a72200-ad89-11eb-9396-8c876be6b309.png)